changelog_page.php

```diff
@@ -35,28 +35,31 @@
  * @param int $p_version_id a valid version id
  * @return null
  */
-function print_version_header( $p_version_id ) {
-	$t_project_id   = version_get_field( $p_version_id, 'project_id' );
-	$t_version_name = version_get_field( $p_version_id, 'version' );
-	$t_project_name = project_get_field( $t_project_id, 'name' );
-
-	$t_release_title = '<a href="changelog_page.php?project_id=' . $t_project_id . '">' . string_display_line( $t_project_name ) . '</a> - <a href="changelog_page.php?version_id=' . $p_version_id . '">' . string_display_line( $t_version_name ) . '</a>';
-
-	if ( config_get( 'show_changelog_dates' ) ) {
-		$t_version_released = version_get_field( $p_version_id, 'released' );
-		$t_release_timestamp = version_get_field( $p_version_id, 'date_order' );
-
-		if ( (bool) $t_version_released ) {
-			$t_release_date = ' (' . lang_get('released') . ' ' . string_display_line( date( config_get( 'short_date_format' ), $t_release_timestamp ) ) . ')';
-		} else {
-			$t_release_date = ' (' . lang_get( 'not_released' ) . ')';
-		}
-	} else {
-		$t_release_date = '';
-	}
-
-	echo '<br />', $t_release_title, $t_release_date, lang_get( 'word_separator' ), print_bracket_link( 'view_all_set.php?type=1&temporary=y&' . FILTER_PROPERTY_PROJECT_ID . '=' . $t_project_id . '&' . filter_encode_field_and_value( FILTER_PROPERTY_FIXED_IN_VERSION, $t_version_name ), lang_get( 'view_bugs_link' ) ), '<br />';
-
+function print_version_header( $p_project_id, $p_version_id ) {
```

Comment:
Update function description with the new @param. Also version_id seems to be optional now.